bundles/org.openhab.binding.miele/src/main/resources/OH-INF/thing/coffeemachine.xml

```diff
@@ -28,6 +28,15 @@
 				<description>The identifier identifies one certain appliance on the ZigBee network.</description>
 				<required>true</required>
 			</parameter>
+			<parameter name="protocol" type="text">
+                <label>Protocol</label>
+                <description>The protocol used to communicate with the appliance</description>
+                <required>true</required>
```

Comment:
For backward compatibility reasons, might it make sense to use ZigBee as default and make this parameter not required anymore?

Reply:
Fair enough :-) BTW, I am not sure Miele still supports/sells ZigBee, from that perspective LAN might need to be the default choice, no?

Reply:
If those devices were only added together with the WLAN support and haven't been there for the previous Zigbee support, I'd agree - you will probably know better than I :-)